### ISSUE TITLE
Continue search for "Open symbol by name" in background

### ISSUE THREAD

**Issue by ToBoMi** [microsoft/vscode#184053]
When the [Open symbol by name](https://code.visualstudio.com/Docs/editor/editingevolved#_open-symbol-by-name) feature is used in a bigger project the search may take longer.
While this search is ongoing one might want to do something else in vscode or even in some other program.

When switching the focus to the vscode explorer pane or even another program the symbol search closes and stops.
This means for now that one has to wait for the search to finish and chose the symbol if it was found. Otherwise one has to repeat that step.

![image](https://github.com/microsoft/vscode/assets/64197724/f0294ed0-f6b3-482f-8066-6d83b393b6ae)

Can this search be changed to also run in background?

**Comment by bpasero** [comment]
Configure `workbench.quickOpen.closeOnFocusLost` in settings.

**Comment by ToBoMi** [comment]
Reading this answer makes me feel not been taken seriously.
The setting you suggest helps in the specific case of this issue, but it also keeps the window open when one wants to close it by changing the focus to something else. If one wants to make things better it should be done ideally without making things worse somewhere else.
I would like to hear that there is no real solution (that's fine) rather than some workaround - or declare a reply as workaround.
That would be a better feeling I guess.

I want to suggest a solution I can think of:

- keep "quickOpen" open in background (like closing the panel, but showing an ongoing search in the statusbar) if there is an ongoing search process and the focus is lost
  - The background search process can be stopped by opening the panel again (clicking on the status bar entry or using CTRL-P) and then using a panel decoration icon (to be implemented) like the one to stop searching for text in files to stop the search
- if the background process stopped (finished - with or without results) a notification could pop up to indicate that and to link to that activity (click on notification opens the panel again)

This morning I tried your suggestion and while I did that the search performs much quicker than in the past (in fact the results are immediately there) - which means I cannot reproduce this issue and since it seems I am the only one complaining about this currently then I would leave this issue closed for now.
I would like to come back if this issue pops up again.
